lib/application/index.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @fileOverview
+ * A module that provides a minimalist application class, useful for managing
+ * configuration and rendering.
+ */
+
+// --------------------------------------------------------------------------
+// Functions.
+// --------------------------------------------------------------------------
+
+/**
+ * Process a queue.
+ *
+ * @param {Mixed[]} queue An array of queued items.
```

Comment:
Super tiny nitpick (seriously, on a scale from 0 to 10, this is a 0.00001): if we'd like to keep with GCC's annotation, `Mixed[]` should be replaced with `Array`, according to [Google's type expressions](https://developers.google.com/closure/compiler/docs/js-for-compiler?hl=en#types).
Then again, if we're not at all concerned with GCC annotation, then I'm not at all worried about it.